### DISCUSSION TITLE
How to run JSON workflow using API

### DISCUSSION THREAD

**Discussion by ksharma24727** [elsa-workflows/elsa-core#3960] [Q&A]
We have created Workflow JSON using some custom logic, Now we want to execute it using some API endpoint like we can execute workflows created from ELSA dashboard(Database Workflows) using API endpoint http://baseURL/workflow-name/execute.

![image](https://user-images.githubusercontent.com/131878242/234576726-5ae8769b-6b3e-4ec9-b5fb-d2435699ba5b.png)

we have uploaded our JSON workflows like above using BlobStorageWorkflowProviderOptions.

Is there anyway in which we can execute our JSON workflows loaded from blob storage Using some API end point.

**Comment by sfmskywalker** [comment]
Yes, you use the same API endpoint to execute workflows provided from JSON files. I assume you have tried it but then it didn't work? If so, can you share what you tried, exactly?
